packages/components/src/Table/Header/TableHeader.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import theme from './TableHeader.scss';
+import TableHeaderCell from './TableHeaderCell';
+
+function getHeaderClassName(classnames) {
+	if (classnames && classnames.header) {
+		return classnames.header;
+	}
+	return null;
+}
+
+function getHeaderCellClassName(column) {
```

Comment:
You don't need this function, you can directly write `column.headClassName`. (I removed `column.key`, it's weird adding that as classname)